src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -104,7 +104,18 @@ public WindowImpl()
             _framebuffer = new FramebufferManager(_hwnd);
 
             if (Win32GlManager.EglFeature != null)
-                _gl = new EglGlPlatformSurface(Win32GlManager.EglFeature.DeferredContext, this);
+            {                
```

Comment:
TODO check from windows 10 version,, and instantiate the compatible version.... also check dx version.

Reply:
done.